scripts/delete_vm.py

```diff
@@ -89,8 +88,7 @@ def rhevm_vm_status(api, vm_name):
 
 
 def vsphere_vm_status(api, vm_name):
-    vm = VirtualMachine.get(api.api, name=vm_name)
-    return vm.runtime.powerState
+    return api.vm_status(vm_name)
 
 
 def rhevm_is_vm_up(api, vm_name):
```

Comment:
**QUESTION**

can we replace all of those with the is_vm_running/is_vm_stopped wrapanapic alls (i just took a look and noted the bascia apis do this more correct)